spec/integ/admin-rooms.spec.js

```diff
@@ -593,6 +647,53 @@ describe("Admin rooms", function() {
         expect(sentAckNotice).toBe(true, "sent ACK m.notice");
     }));
 
+    it("should not try to change to a nickname that is already in use", async () => {
+        const newNick = "Blurple";
+
+        // make sure that the NICK command not is sent
+        let sentNickCommand = false;
+        env.ircMock._whenClient(roomMapping.server, userIdNick, "send",
+        function() {
+            sentNickCommand = false;
```

Comment:
```suggestion
            sentNickCommand = true;
```

Reply:
We shouldn't be sending a nickname request if we know the nick is in use, as per the test title?

Reply:
oh...oh....ah